include/linux/mptcp_upstream.h

```diff
@@ -84,6 +86,8 @@ enum {
 	MPTCP_PM_CMD_SET_LIMITS,
 	MPTCP_PM_CMD_GET_LIMITS,
 	MPTCP_PM_CMD_SET_FLAGS,
+        MPTCP_PM_CMD_ANNOUNCE,
+        MPTCP_PM_CMD_REMOVE,
```

Comment:
I know there's more to the new API than this.  More to come in a subsequent pull request.